apps/sample-app/src/App.tsx

```diff
@@ -20,12 +29,19 @@ const App: React.FC = () => {
         app.notifyAppLoaded();
         app.notifySuccess();
         const context = await app.getContext();
-        const themeNow = getTheme(context?.app?.theme);
-        setCurrTheme(themeNow);
-        app.registerOnThemeChangeHandler(function(theme) {
-          setCurrTheme(getTheme(theme));
-        });
+        if (context?.app?.host?.name === 'Teams') {
```

Comment:
I think we still want to avoid checking for specific hosts, right?

Reply:
I implement this so we can have the right host theme for the app. I think if we want the app to adjust to its host's theme we would have to check the host.name?